server/api/person/person.subscribe.js

```diff
@@ -24,4 +25,25 @@ module.exports = (server) => {
     })
     PubSub.publish(TOPIC_PERSON__EMAIL_SENT, info)
   })
+
+  PubSub.subscribe(TOPIC_MEMBER__UPDATE, async (msg, member) => {
```

Comment:
Should member specific subscriptions live in `server/api/member/member.subscribe.js`?

Reply:
I thought about that.  we put the subs where they are relevant to be handled rather than started. So while the publish might be in member or interest. the subs for adding goal cards are in personalGoals, and the subs for sending emails to people are in person.  If we added a transaction log that would have its own module and collect any topics it needed to log.